src/content/blog/astro-210.mdx

```diff
@@ -0,0 +1,169 @@
+- **[TypeScript helpers for getStaticPaths](#typescript-helpers-for-getstaticpaths)**: Get the params and props without defining the types twice.
+
+Install this new release by running `npm install astro@latest`.
+
+## Image support in Astro core
+
+Image optimizations are something almost every website needs. That's why we're taking what we learned through the `@astrojs/image` integration and bringing that into core where image optimizations are more tightly integrated with other Astro features such as [Content Collections](https://docs.astro.build/en/guides/content-collections/).
+
+The following features are available behind the new `--experimental-assets` flag. We'll continue to revise and improve support for images until the feature is stable and the flag dropped in a future release. You can contribute your feedback by commenting on the [image RFC](https://github.com/withastro/roadmap/pull/500).
+
+### Assets folder
+
+The new `src/assets/` folder is the recommended place to put images. You can then access them through a newly provided alias:
+
+```astro
+---
+import penguin from "~/assets/penguin.png"
+---
+```
+
+Images in this folder are also available to be optimized through content collections (explained below).
+
+### Image component
+
+The `<Image />` component has been migrated from the `@astrojs/image` integration. A typical usage of the image component looks like this:
+
+```astro
+---
+import { Image } from "astro:asset"
+import penguin from "~/assets/penguin.png"
+---
+
+<Image src={penguin} quality="high" />
+```
+
+As with the integration, you can adjust the size of the image using the `width` and `height` attributes. If not provided the image will be optimized for its actual dimensions. Additionally, the new `quality` attribute allows for adjusting the quality scale of the image.
+
+### Content Collections usage
+
+A new `image()` helper is used to define a property on your schema as being an image. Provided that the image is in the `src/assets/` folder, it will be automatically checked for you.
+
+```ts
+import { image, defineCollection, z } from "astro:content"
+
+const blogCollection = defineCollection({
+	schema: z.object({
+		title: z.string(),
+		cover: image().refine((img) => img.width === 600, {
+			message: "Cover needs to be 600 pixels wide!",
+		}),
+	}),
+})
+
+export const collections = {
+	blog: blogCollection,
+}
+```
+
+And then provide this property in any of your items in the collection:
+
+```md
+---
+title: Penguins are cool
+image: emperor-penguin.png
+---
+
+# Migration patterns of emperor penguins
+
+Post here...
+```
+
+Using this schema is a great way to ensure that your images in a collection are all of the same dimensions.
+
+## Markdoc integration
+
+<BlogContentImage src={markdoc} alt="The logo for the Markdoc project" />
+
+As we look for ways to scale Astro sites to hundreds of thousands of pages, we are continuously looking for ways to make our build times faster.
```

Comment:
for an announcement post like this, I think it's better to get right to the info in the next paragraph. Either remove this, or merge it into the content below somehow